### ISSUE TITLE
Uploading profile image - need to give error warning when it does not upload because it is too large.

### ISSUE THREAD

**Issue by corinnewo** [ji-devs/ji-cloud#2945]
Chana tried to upload an image that is 3.5MB. Nothing happened, and the image did not upload. When she tried a smaller image, it uploaded perfectly.
There must be an error message if the file size is too large, or if the file type is wrong.

Error message, if file is too large 
"**Oh no! This file is too heavy.
Maximum file size: x MB**" _(where x is the actual limit on the system)_

Error message, if file is not an acceptable image file  (@ChaykeeMor please edit..)
"Oh no! We can't load that type of file.
Try uploading an image file, like png or jpg."

**Comment by ChaykeeMor** [comment]
We need to offer real help with warnings. What IS the max file size? What file formats CAN we use? I have asked these questions several times and I either get a wrong answer or no answer. Do we know? Or should I just discover through trial and error? @corinnewo

**Comment by corinnewo** [comment]
@ChaykeeMor I think that the max file size depends on where you are. When uploading a resource, we discussed limiting to 10MB (#2185), but I see that it has not actually been fixed yet. I think with the profile image, it certainly should be much less. No need to have an image that is many MBs. That's why I have used the [x]... should definitely be more defined and helpful. For profile image, should be an image file, but @MendyBerger will have to give the list of what is permissible. Definitely, we can recommend png or jpg.
